browser/brave_profile_prefs.cc

```diff
@@ -390,6 +391,8 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   speedreader::SpeedreaderService::RegisterProfilePrefs(registry);
 #endif
 
+  de_amp::DeAmpUtil::RegisterProfilePrefs(registry);
```

Comment:
nit: convention for this would be `de_amp::RegisterProfilePrefs` like `safe_browsing::RegisterProfilePrefs`